apps/ewallet/lib/ewallet/web/paginator.ex

```diff
@@ -102,10 +184,49 @@ defmodule EWallet.Web.Paginator do
 
   @doc """
   Paginate a query using the given `page` and `per_page` and returns a paginator.
+  If a query has `page_record_value`, then returns a paginator with all records after the specify `page_record_value`.
   """
-  def paginate(queryable, page, per_page, repo \\ Repo) do
-    {records, more_page} = fetch(queryable, page, per_page, repo)
+  def paginate(queryable, attrs, repo \\ Repo)
+
+  def paginate(queryable, %{"page" => _, "per_page" => _} = attrs, repo) do
+    {records, more_page} = fetch(queryable, attrs, repo)
+
+    # Return pagination result
+    paginate(attrs, more_page, records)
+  end
+
+  def paginate(
+        queryable,
+        %{
+          "page_record_field" => page_record_field,
+          "page_record_value" => page_record_value,
+          "per_page" => per_page
+        } = attrs,
+        repo
+      ) do
+    # Verify if the given `page_record_value` exist to prevent unexpected result.
+    if repo.get_by(queryable, id: page_record_value) == nil do
```

Comment:
An if/else clause should not cover a large portion of the logic. In this case it covers the whole function.

Can you extract this into a separate pattern matched function?